domain-xample/xample-workflows/src/main/java/gov/va/vro/routes/xample/BieXampleRoutes.java

```diff
@@ -14,6 +17,11 @@ public class BieXampleRoutes {
   private final DbHelper dbHelper;
   private final ObjectMapper objectMapper;
 
+  final IMetricLoggerService metricLogger = new MetricLoggerService();
```

Comment:
Don't do it if you aren't interested, but I would encourage using spring's dependency injection magic for `MetricLoggerService` instead of direct instantiation. Everything surrounding this BieXampleRoutes class is already wired for it. For example dbHelper and objectMapper are both getting injected, so that is a good clue that this class is already doing that.  I believe all that is needed in this case is to remove the "= new MetricLoggerService()" and then just ensure that the MetricLoggerService is decorated with @Service.  This could be something to play around with if you are looking to improve familiarity with spring and dependency injections.  The next trick would be to figure out how toggle the implementation used based on the LocalEnvironmentCondition.

Reply:
Seconding using dependency injection for a MetricLoggerService bean specifically but I'd argue it's a required change - a bean already exists and by creating a new instance here we're recreating the backend connections to DataDog, but with the default MetricApi/ApiClient configuration instead of whatever is defined in DatadogClientConfig.java.

Reply:
1. Spring has been a weak spot for me; I get nervous with dependency injection!
2. We should be using dependency injection in this case, agreed.
3. Addressed in 0ca7048.